test/livebook/intellisense_test.exs

```diff
@@ -3,22 +3,62 @@ defmodule Livebook.IntellisenseTest do
 
   alias Livebook.Intellisense
 
+  @tmp_dir "tmp/test/remote_intellisense"
+
   # Returns intellisense context resulting from evaluating
   # the given block of code in a fresh context.
-  defmacrop eval(do: block) do
+  defmacrop eval(node \\ nil, do: block) do
     quote do
       block = unquote(Macro.escape(block))
       binding = []
       env = Code.env_for_eval([])
       {value, binding, env} = Code.eval_quoted_with_env(block, binding, env)
+      node = unquote(node) || node()
 
       %{
         env: env,
-        map_binding: fn fun -> fun.(binding) end
+        map_binding: fn fun -> fun.(binding) end,
+        node: node
       }
     end
   end
 
+  setup_all do
+    {:ok, _pid, node} =
+      :peer.start(%{
+        name: :remote_runtime,
+        args: [~c"-setcookie", Atom.to_charlist(Node.get_cookie())]
+      })
+
+    {:module, Elixir.RemoteModule, bytecode, _} =
+      defmodule Elixir.RemoteModule do
+        @compile {:autoload, false}
+        @moduledoc """
+        Remote module docs
+        """
+
+        @doc """
+        Hello doc
+        """
+        def hello(message) do
+          message
+        end
+      end
+
+    false = Code.ensure_loaded?(RemoteModule)
+
+    File.rm_rf!(@tmp_dir)
+    File.mkdir_p!(@tmp_dir)
+    File.write!("#{@tmp_dir}/Elixir.RemoteModule.beam", bytecode)
+
+    true = :erpc.call(node, :code, :set_path, [:code.get_path() ++ [~c"#{@tmp_dir}"]])
```

Comment:
```suggestion
    true = :erpc.call(node, :code, :set_path, [[~c"#{@tmp_dir}" | :code.get_path()]])
```

Reply:
Actually, let's prune this path a bit:

```suggestion
    build_path = Mix.Project.build_path() |> String.to_charlist()
    paths = Enum.reject(:code.get_path, &List.starts_with?(&1, build_path))
    true = :erpc.call(node, :code, :set_path, [[~c"#{@tmp_dir}" | paths]])
```

This will make sure the node does not have any of our deps either!